airflow/www/templates/airflow/dags.html

```diff
@@ -369,14 +369,15 @@ <h2>DAGs</h2>
     function lastDagRunsHandler(error, json) {
```

Comment:
at this point this function name needs changing too

Reply:
Well, I am not adding new functionality, just fixing a bug/regression. I am just putting back a value lost when the value was changed to load asynchronously. But it is a valid point as that function was added at that point it was change to asynchronous.

This function appears to be named the same as the `/last_dagruns` endpoint that returns json for these dates, which is in turn seems to be named after the "Last Run" column this tooltip is in (I think). So the column is showing the last run's execution date and the last run's start date. What name should it be instead?

Reply:
Correction, the function is named after the `dag.get_last_dagrun` method which was used in the pre-asynchronous code for the value of `last_run`, which then was used for the values `last_run.execution_date` and `last_run.start_date`, but was incorrectly changed to be last_run = execution_date when it went asynchronous. This PR just puts the values back to as they were for the asynchronous code.